index.css

```diff
@@ -0,0 +1,154 @@
+@media (min-width: 400px) {
+  .header_container {
+    position: absolute;
+    width: 375px;
+    height: 92px;
+
+    /* left: 0px;
+    top: 0px; */
+
+    background: #3c3a39;
+    color: red;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .item {
+    align-self: center;
+  }
+
+  .name {
+    font-family: Roboto, serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .menu_bar {
+    width: 24px;
+    height: 24px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .menu_bar img {
+    width: 18px;
+    height: 12px;
+    left: 3px;
+    top: 6px;
+  }
+
+  .main_container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 120px 24px;
+    gap: 12px;
+    position: absolute;
+    width: 375px;
+    height: 590px;
+    top: 92px;
+    background: #1c1a19;
+    background-image: url('img/Illustration-header.png');
```

Comment:
As I mentioned earlier, please check your bg works fine on 375px mobile view. This is important to have an aesthetic portfolio.